### DISCUSSION TITLE
Datahub, Add Configuration disabled

### DISCUSSION THREAD

**Discussion by incoxx** [pimcore/pimcore#10940] [Q&A / Support] [answered]
Hello,

i am using PIMCORE 10.2.5 and did an installation of the datahub:

https://pimcore.com/docs/data-hub/current/Installation.html

![grafik](https://user-images.githubusercontent.com/21988262/144400443-9cb90a76-4044-4e36-9509-08778467ce05.png)

No problem with the installation. But when i open the Datahub icon "Add configuration" is disabled

![grafik](https://user-images.githubusercontent.com/21988262/144400175-ef924f8a-02ce-4d46-ab24-d8ff1645c409.png)

Any idea? Thank you! Markus

**Comment by fashxp** [comment, marked as answer]
see https://pimcore.com/docs/pimcore/current/Development_Documentation/Deployment/Configuration_Environments.html#page_Production-environment-with ...

in production environments, you need to configure `settings-store` as configuration write target.

**Reply by incoxx** [reply]
Hello Christian, thx!

Final question: Is there a specific env variable for the Datahub (?)

![grafik](https://user-images.githubusercontent.com/21988262/144407672-fca896d3-05e2-4c43-b02a-0e6324a9f142.png)

Thank you! Markus

**Reply by incoxx** [reply]
I am sorry ! Got it PIMCORE_WRITE_TARGET_DATA_HUB

**Comment by incoxx** [comment]
It works :-)

If somebody has the same problem on a PROD environment:

See:
https://pimcore.com/docs/data-hub/current/Deployment.html
https://pimcore.com/docs/pimcore/current/Development_Documentation/Deployment/Configuration_Environments.html

"My" .env file in the pimcore root folder:

![grafik](https://user-images.githubusercontent.com/21988262/144410011-7b460c84-8b50-4b7d-bc6e-c21504513f58.png)

**Reply by fashxp** [reply]
thx for sharing, maybe we can add a note to datahub docs somewhere here: https://pimcore.com/docs/data-hub/1.1/Deployment.html#page_Configuration-Storage
